Dev build - missing instructions

Hello,

This project seems super interesting, and I've managed to quickly try it. I wanted to try and write some code for it, but I am unfamiliar with go.

The "make" command complained about missing packages, so I ran:

```
$ go get github.com/spf13/cobra/doc
$ go get github.com/MichaelMure/git-bug/commands
```

Now, make fails with another error:

```
$ make
go generate
# command-line-arguments
doc/gen_markdown.go:20:28: cannot use commands.RootCmd (type *"github.com/MichaelMure/git-bug/vendor/github.com/spf13/cobra".Command) as type *"github.com/spf13/cobra".Command in argument to doc.GenMarkdownTree
git-bug.go:1: running "go": exit status 2
make: *** [Makefile:4: build] Error 1
```


I feel like README.md should have specific instructions for developers. Experienced Golang devs will not learn much, but it would be really helpful for people like me.

--- a/README.md
+++ b/README.md
@@ -118,6 +118,20 @@ Interested by how it works ? Have a look at the [data model](doc/model.md).
 
 ## Contribute
 
+Developers unfamiliar with Go may try to clone the repository using "git clone".
+Instead, one should use:
+
+```shell
+go get -u github.com/MichaelMure/git-bug
+```
+
+The git repository will then be available:
+
+```shell
+# Note that $GOPATH defaults to $HOME/go
+$ cd $GOPATH/src/github.com/MichaelMure/git-bug/
+```
+
 PRs accepted. Drop by the [Gitter lobby](https://gitter.im/the-git-bug/Lobby) for a chat.
 
 ## Contributors

